Gpu optimisation

will-44 opened the pull request:
Use a point cloud to display the reachability map instead of markers. This is use for RM with 0.02m resolution

Commit:
transfer to ros2 working !

Commit:
load rm just from attribute and data

will-44 committed:
read new h5 file structure

will-44 committed:
publish voxel grid as marker

will-44 committed:
oups hpp

will-44 committed:
to present many results

will-44 committed:
change for pcd

will-44 committed:
interface working to viz !

will-44 committed:
clean

will-44 committed:
change for big pcd

will-44 committed:
clear warning and compatible humble jazzy

will-44 reviewed (commented):
oups

will-44 committed:
visualisation en focntion de l'intensité

will-44 committed:
Merge branch 'gpu_optimisation' of https://github.com/Lab-CORO/reachability_map_visualizer into gpu_optimisation

will-44 committed:
visualisation avec intensité jetmap

will-44 committed:
utilisation d'un  timer et non d'une loop (et class)

will-44 commented:
Hello,
Thank you for your interest in my PR. I actually made a mistake — I only intended to open a PR on my own fork.
I'm currently working on a better implementation to achieve higher frequency and resolution, so I will open a new PR with documentation in the coming weeks.

I'm a PhD student in Montreal in the CoRo Lab, working on a real-time reachability map. I read your article and was wondering if you would be interested in having a meeting to discuss our work.
I already have some preliminary results, but I'm still thinking about the next steps.

Thanks again for your time!

sjauhri commented:
Hi Will,

Sure, I'm happy to meet and discuss.

Regards,

Snehal Jauhri


On Fri, 14 Nov 2025 at 15:53, will-44 ***@***.***> wrote:

> *will-44* left a comment (pearl-robot-lab/reachability_map_visualizer#3)
> <https://github.com/pearl-robot-lab/reachability_map_visualizer/pull/3#issuecomment-3533149550>
>
> Hello,
> Thank you for your interest in my PR. I actually made a mistake — I only
> intended to open a PR on my own fork.
> I'm currently working on a better implementation to achieve higher
> frequency and resolution, so I will open a new PR with documentation in the
> coming weeks.
>
> I'm a PhD student in Montreal in the CoRo Lab, working on a real-time
> reachability map. I read your article and was wondering if you would be
> interested in having a meeting to discuss our work.
> I already have some preliminary results, but I'm still thinking about the
> next steps.
>
> Thanks again for your time!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pearl-robot-lab/reachability_map_visualizer/pull/3#issuecomment-3533149550>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKKKG4KWLANOW2TLAA3UVND34XUGBAVCNFSM6AAAAACKLLT4C6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTKMZTGE2DSNJVGA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> .com>
>

will-44 committed:
Add README and enhance reachability map visualization features

- Introduced README.md for project documentation, detailing dependencies, build instructions, usage, node parameters, topics, and RViz plugin properties.
- Updated load_reachability_voxel.cpp to declare parameters for HDF5 path and frame ID.
- Enhanced reachability_map_display.cpp and reachability_map_display.h to include map index property and functionality for updating the index.
